lib/src/postgres_target.rs

```diff
@@ -362,6 +298,9 @@ pub async fn init(
     async_channel::Sender<AccountWrite>,
     async_channel::Sender<SlotUpdate>,
 )> {
+    // Retrieve up to batch_size account writes
+    let mut write_batch = Vec::new();
```

Comment:
why is this vector moved to outside of the tokio thread?

Reply:
I see why, but I think we shouldn't need this. IN the slot update thread we only need the slots.